### PR TITLE
CONFIG_MPSL_CX disabled by default

### PR DESCRIPTION
**Pull request by jciupis**

### CODE REVIEW

**Review by bama-nordic — APPROVED** [review]
LGTM

**Review by wentong-li — APPROVED** [review]

**Review by muraliThokala — APPROVED** [review]

File: doc/nrf/app_dev/wifi_coex/index.rst [hunk 1]
```diff
@@ -219,9 +219,7 @@ The nRF700x interface is supported out of the box.
 To use it, complete the following steps:
 
 1. Add the devicetree `nrf_radio_coex` node as described in :ref:`ug_radio_cx_nrf700x_hw_if`.
-2. If :kconfig:option:`CONFIG_WIFI` is set to ``y`` and one of the supported SR protocols is enabled no additional steps are needed.
-   If you do not wish to use the radio coexistence implementation, you must set  :kconfig:option:`CONFIG_MPSL_CX` to ``n``.
-3. If :kconfig:option:`CONFIG_WIFI` is disabled or none of the supported SR protocols are enabled, the implementation must be enabled manually with :kconfig:option:`CONFIG_MPSL_CX` and :kconfig:option:`CONFIG_MPSL_CX_NRF700X`.
+2. Set the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option to ``y``. On the nRF5340, you must apply the Kconfig option also to the network core.
```
Comment (FrancescoSer) [inline doc/nrf/app_dev/wifi_coex/index.rst:222]:
```suggestion
#. Set the :kconfig:option:`CONFIG_MPSL_CX` Kconfig option to ``y``.
   On the nRF5340, you must apply the Kconfig option also to the network core.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -170,6 +170,10 @@ Wi-Fi
 
   * nRF7002 revision A support.
 
+* Updated:
+
+  * WiFi Coexistence is no longer enabled by default. It must be enabled explicitly in Kconfig by setting :kconfig:option:`CONFIG_MPSL_CX`. In case of nRF5340, this option must be selected for both application core and network core images.
```
Comment (FrancescoSer) [inline doc/nrf/releases/release-notes-changelog.rst:175]:
```suggestion
  * WiFi Coexistence is no longer enabled by default.
    It must be enabled explicitly in Kconfig using :kconfig:option:`CONFIG_MPSL_CX`.
    On the nRF5340, this option must be selected for both the application core and the network core images.
```

**Review by FrancescoSer — APPROVED** [review]
Approving but please implement these 2 suggestions

**Pull request merged**